server/sonar-server/src/main/java/org/sonar/server/ce/ws/ActivityAction.java

```diff
@@ -298,4 +295,159 @@ private static ActivityRequest toSearchWsRequest(Request request) {
       PARAM_COMPONENT_ID, PARAM_COMPONENT_QUERY);
     return activityWsRequest;
   }
+
+  private static class Request {
+
+    private String componentId;
+    private String componentQuery;
+    private String maxExecutedAt;
+    private String minSubmittedAt;
+    private String onlyCurrents;
+    private String p;
+    private String ps;
+    private String q;
+    private List<String> status;
+    private String type;
+
+    /**
+     * Example value: "AU-TpxcA-iU5OvuD2FL0"
+     */
+    public Request setComponentId(String componentId) {
+      this.componentId = componentId;
+      return this;
+    }
+
+    public String getComponentId() {
+      return componentId;
+    }
+
+    /**
+     * Example value: "Apache"
+     * @deprecated since 5.5
+     */
+    @Deprecated
+    public Request setComponentQuery(String componentQuery) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)